src/code/z_rumble.c

```diff
@@ -45,11 +54,11 @@ void Rumble_Request(f32 distSq, u8 arg1, u8 ticksTimer, u8 decreaseStep) {
         temp = arg1 - (distance * 255) / 1000;
 
         for (i = 0; i < RUMBLE_REQUEST_BUFFER_SIZE; i++) {
-            if (gRumbleMgr.unk_04[i] == 0) {
+            if (gRumbleMgr.requestIntensities[i] == 0) {
                 if (temp > 0) {
-                    gRumbleMgr.unk_04[i] = temp;
-                    gRumbleMgr.timer[i] = ticksTimer;
-                    gRumbleMgr.decreaseStep[i] = decreaseStep;
+                    gRumbleMgr.requestIntensities[i] = temp;
+                    gRumbleMgr.requestDecayTimers[i] = ticksTimer;
+                    gRumbleMgr.requestDecaySteps[i] = decreaseStep;
```

Comment:
I think these (and the names in the cutscene files) should be closer to the RumbleManager names, I'd already forgotten which was which when I was looking at them.